packages/mobile/App.svelte

```diff
@@ -128,78 +128,76 @@
     $appSettings.notifications = false
 </script>
 
-{#if $isLocaleLoaded && !showSplash}
-    <!-- empty div to avoid auto-purge removing dark classes -->
-    <div class="scheme-dark" />
-    <div class="scanner-hide" style="--transition-scroll: cubic-bezier(0, 0.3, 0, 1)">
-        {#if $popupState.active}
-            <Popup
-                type={$popupState.type}
-                props={$popupState.props}
-                hideClose={$popupState.hideClose}
-                fullScreen={$popupState.fullScreen}
-                transition={$popupState.transition}
-                overflow={$popupState.overflow}
-                locale={$_}
-            />
-        {/if}
-        <Route route={AppRoute.Welcome}>
-            <Welcome locale={$_} />
-        </Route>
-        <Route route={AppRoute.Legal}>
-            <Legal locale={$_} />
-        </Route>
-        <Route route={AppRoute.CrashReporting}>
-            <CrashReporting locale={$_} />
-        </Route>
-        <Route route={AppRoute.Appearance}>
-            <Appearance locale={$_} />
-        </Route>
-        <Route route={AppRoute.Profile}>
-            <Profile locale={$_} />
-        </Route>
-        <Route route={AppRoute.Setup}>
-            <Setup locale={$_} />
-        </Route>
-        <Route route={AppRoute.Create}>
-            <Create locale={$_} />
-        </Route>
-        <Route route={AppRoute.Secure}>
-            <Secure locale={$_} />
-        </Route>
-        <Route route={AppRoute.Password}>
-            <Password locale={$_} />
-        </Route>
-        <Route route={AppRoute.Protect} transition={false}>
-            <Protect locale={$_} />
-        </Route>
-        <Route route={AppRoute.Backup} transition={false}>
-            <Backup locale={$_} />
-        </Route>
-        <Route route={AppRoute.Import} transition={false}>
-            <Import locale={$_} />
-        </Route>
-        <Route route={AppRoute.Balance}>
-            <Balance locale={$_} />
-        </Route>
-        <Route route={AppRoute.Migrate}>
-            <Migrate locale={$_} {goto} />
-        </Route>
-        <Route route={AppRoute.Congratulations}>
-            <Congratulations locale={$_} {goto} />
-        </Route>
-        <Route route={AppRoute.Dashboard}>
-            <Dashboard locale={$_} {goto} />
-        </Route>
-        <Route route={AppRoute.Login}>
-            <Login locale={$_} {goto} />
-        </Route>
-        <ToastContainer />
-    </div>
-    <div class="scanner-ui">
-        <QRScanner locale={$_} />
-    </div>
-{/if}
+<!-- empty div to avoid auto-purge removing dark classes -->
```

Comment:
we might need to add here or wrapping the relevant routes a `#key $isLocaleLoaded` because with these removals the localization is not propagating properly when you change language, things dont translate properly until you close the app and start again